cmd/process.go

```diff
@@ -117,19 +118,25 @@ func process(ctx context.Context, logger *zap.SugaredLogger) error {
 		logger.Fatal("failed to initialize new server", zap.Error(err))
 	}
 
+	conn, err := events.NewConnection(config.AppConfig.Events)
+	if err != nil {
+		logger.Fatalw("failed to create new events connection", "error", err)
+	}
+
 	server := &srv.Server{
-		Echo:             eSrv,
-		Chart:            chart,
-		Context:          cx,
-		Debug:            viper.GetBool("logging.debug"),
-		KubeClient:       client,
-		Logger:           logger,
-		EventTopics:      viper.GetStringSlice("event-topics"),
-		ChangeTopics:     viper.GetStringSlice("change-topics"),
-		SubscriberConfig: config.AppConfig.Events.Subscriber,
-		ValuesPath:       viper.GetString("chart-values-path"),
-		Locations:        viper.GetStringSlice("event-locations"),
-		MetricsPort:      viper.GetInt("loadbalancer-metrics-port"),
+		Echo:         eSrv,
+		Chart:        chart,
+		Connection:   conn,
```

Comment:
Name this is something more detailed? Maybe `EventsConnection` or something like that?